docs/source/index.rst

```diff
@@ -1,3 +1,14 @@
+=====================
```

Comment:
Even though release notes documents are usually listed first (RHEL is one example), I'd put this at the end, as the lsat section.  The reason is that we intend to keep the release notes mostly for history and reference purposes, and I'd get in the way of the general user experience the first time they land at our docs page.